var/spack/repos/builtin/packages/py-vermin/package.py

```diff
@@ -37,9 +38,6 @@ class PyVermin(PythonPackage):
     version("0.10.4", sha256="bd765b84679fb3756b26f462d2aab4af3183fb65862520afc1517f6b39dea8bf")
     version("0.10.0", sha256="3458a4d084bba5c95fd7208888aaf0e324a07ee092786ee4e5529f539ab4951f")
 
-    depends_on("python@2.7:", type=("build", "run"))
-    depends_on("py-setuptools", type=("build", "run"))
```

Comment:
It's still needed to build the package though:
```python
depends_on("py-setuptools", type="build")
```
Were there any older versions that required it at run-time?

Reply:
Oh, yeah, my bad. No, I think I incorrectly copied it from another recipe initially.